.bash_aliases

```diff
@@ -50,27 +56,41 @@ function ls-lined-executables() { ls-lined | grep -iE '[*]$' ; }
 function ls-lined-executables-c() { ls-lined-executables | xargs -I {} bash -c 'colorize Green "{}"' ; }
 alias lsc="ls-lined-folders-c && ls-lined-files-c && ls-lined-executables-c"
 
+# VS Code
+alias edit_snippets="$EDIT ~/Library/Application\ Support/Code/User/Snippets/*"
+alias edit_vscode_settings="$EDIT ~/.Library/Application\ Support/Code/User/settings.json"
```

Comment:
interesting, is this in case you want to edit your vscode settings without opening vs code? I usually just have code open and hit `Cmd + ,`